src/Employer/Employer.Web/Filters/CheckEmployerBlockedFilter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading.Tasks;
+using Esfa.Recruit.Employer.Web.Caching;
+using Esfa.Recruit.Employer.Web.Configuration.Routing;
+using Esfa.Recruit.Employer.Web.Exceptions;
+using Esfa.Recruit.Vacancies.Client.Application.Providers;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Esfa.Recruit.Employer.Web.Filters
+{
+    public class CheckEmployerBlockedFilter : IAsyncActionFilter
+    {
+        private const string BlockedEmployersCacheKey = "blockedEmployers";
```

Comment:
Maybe stick this in a CacheKey constant file going forward.

Reply:
👍